docs/runtime/beam-metrics.md

```diff
@@ -0,0 +1,947 @@
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| `beam.cpu.logical_number` | int | The logical thread number [0..n-1] | `1` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `beam.cpu.work` | string | The work performed for this data point. | `gc` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`beam.cpu.work` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `aux` | Time spent handling auxiliary jobs. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `check_io` | Time spent checking for new I/O events. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `emulator` | Time spent executing Erlang processes. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gc` | Time spent doing garbage collection. When extra states are enabled this is the time spent doing non-fullsweep garbage collections. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `other` | Time spent doing unaccounted things. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `port` | Time spent executing ports. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `sleep` | Time spent sleeping. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `alloc` | Time spent managing memory. Without extra states this time is spread out over all other states. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `bif` | Time spent in BIFs. Without extra states this time is part of the emulator state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `busy_wait` | Time spent busy waiting. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `ets` | Time spent executing ETS BIFs. Without extra states this time is part of the emulator state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `gc_full` | Time spent doing fullsweep garbage collection. Without extra states this time is part of the gc state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `nif` | Time spent in NIFs. Without extra states this time is part of the emulator state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `send` | Time spent sending messages (processes only). Without extra states this time is part of the emulator state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `timers` | Time spent managing timers. Without extra states this time is part of the other state. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
```

Comment:
We should probably repeat the microstate accounting caveat that seems to only exist in the declaration yaml and code for this section?

Reply:
^ this